src/BHModels.jl

```diff
@@ -1,12 +1,45 @@
+"""
+    arbitraryEjectaBH(m2_i, frac)
+
+#TODO Description
+#RTW do we really need this function? It seems like overkill for such a simple operation
+
+# Arguments:
+#TODO
+- m2_i:
+- frac:
+
+# Output:
+#TODO
+- 
+"""
 function arbitraryEjectaBH(m2_i, frac)
     return m2_i*(1-frac)
 end
 
+"""
+    restrictedEjectaBH(m2_i, frac; Ma=10, Mb=15, max_frac=1.0, min_frac=0.1)
```

Comment:
Ahh, yeah, forgot this was already in. This is what I mean with the previous point. This function essentially avoids black holes above 15 Msun to lose significant mass when considering inference. This is akin to assuming that at those masses the orbital properties must be a consequence of the momentum kick and not a Blaauw kick.